app/src/main/java/org/p2p/wallet/svl/model/SendLinkGenerator.kt

```diff
@@ -6,11 +6,11 @@ import org.p2p.solanaj.core.Account
 import org.p2p.solanaj.crypto.DerivationPath
 import org.p2p.wallet.BuildConfig
 import org.p2p.wallet.newsend.model.TemporaryAccount
+import org.p2p.wallet.svl.interactor.SendViaLinkWrapper
 import org.p2p.wallet.utils.emptyString
 
-private const val REGEX_LINK_ALLOWED_SYMBOLS = "[A-Za-z0-9_~-]"
-private const val ASCII_CHARACTERS_COUNT = 128
-private const val SYMBOLS_COUNT = 12
+private const val LINK_ALLOWED_SYMBOLS = "!$'()*+,-.0123456789@ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz~"
```

Comment:
why? [A-Za-z0-9_~-] works okay

Reply:
It didn't work and we needed to send a build urgently. 
So temporary I reverted back that. 
The algorithm returns not the symbols but a single integer, like 0 or 6.
And thus the link is t.key.app/0 

So I'll need more time to debug it